ext/curl/curl_string_file.stub.php

```diff
@@ -0,0 +1,15 @@
+<?php
+
+/**
+ * @generate-function-entries
+ * @generate-class-entries
+ */
+
+class CURLStringFile
+{
+    public string $data = "";
+    public string $postname = "";
+    public ?string $mime = null;
+
+    public function __construct(string $data, string $postname, ?string $mime = null) {}
```

Comment:
Not directly related: @kocsismate I just noticed that https://github.com/php/php-src/blob/8c00c919e38ea9189745fb49da13df06077df21b/ext/curl/curl_file.stub.php#L17 uses different names in the constructor from the property names. We actually managed to have three different names between $postname, $posted_filename and getPostFilename()...

Reply:
Uhh 🤦